bake/compose.go

```diff
@@ -229,7 +230,9 @@ func composeToBuildkitSecret(inp compose.ServiceSecretConfig, psecret compose.Se
 	}
 
 	var bkattrs []string
-	if inp.Source != "" {
+	if psecret.Environment != "" {
+		bkattrs = append(bkattrs, "id="+psecret.Environment)
```

Comment:
Looks like this should use `env=` (and leave id as-is?)

Reply:
Can be [one-of `id` or `env` right](https://github.com/docker/buildx/blob/master/docs/reference/buildx_build.md#env)? I can change to `env` ofc if it makes more sense.